src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,76 @@
+package lotto.domain;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import static java.util.stream.Collectors.toList;
+
+public class Lotto {
+    private static final String VALID_SIZE_MESSAGE = "로또는 6개의 번호여야 한다.";
+    private static final String VALID_EMPTY_MESSAGE = "값이 없습니다.";
+    private static final String VALID_DUPLICATE_MESSAGE = "로또 번호는 중복이 불가합니다.";
+    private static final int LOTTO_SIZE = 6;
+
+    private final List<LottoNumber> lotto;
+
+    public Lotto(List<LottoNumber> lotto) {
+        validateLotto(lotto);
+        this.lotto = lotto;
+    }
+
+    public static List<LottoNumber> of(List<Integer> lotto) {
+        return lotto.stream()
+                .map(number -> new LottoNumber(number))
+                .collect(toList());
+    }
+
+    private void validateLotto(List<LottoNumber> lotto) {
+        validateEmpty(lotto);
+        validateSize(lotto);
+        validateDuplicate(lotto);
+    }
+
+    private void validateEmpty(List<LottoNumber> lotto) {
+        if (lotto == null || lotto.size() == 0) {
```

Comment:
자바에서 제공하는 메서드를 활용하면 의도를 더 명확히 드러낼 수 있어요!

```suggestion
        if (lotto == null || lotto.isEmpty()) {
```